content/docs/components-and-props.md

```diff
@@ -154,11 +154,11 @@ This component can be tricky to change because of all the nesting, and it is als
 First, we will extract `Avatar`:
 
 ```js{3-6}
-function Avatar(props) {
+function Avatar({user}) {
```

Comment:
We have not used de structuring in the page already? There are other places where we usually don't use destructuring as well.